mslib/msui/editor.py

```diff
@@ -29,11 +29,234 @@
 import json
 
 from mslib.msui.mss_qt import get_open_filename, get_save_filename
+from mslib.msui.mss_qt import ui_configuration_editor_window as ui_conf
 from PyQt5 import QtWidgets, QtGui, QtCore, QtPrintSupport
 from mslib.msui import constants
 from mslib.msui.constants import MSS_CONFIG_PATH, MSS_SETTINGS
 from mslib.msui.icons import icons
 
+from mslib.support.qt_json_view import model
+from mslib.support.qt_json_view.model import JsonModel
+from mslib.support.qt_json_view.view import JsonView
+
+
+class ConfigurationEditorWindow(QtWidgets.QMainWindow, ui_conf.Ui_ConfigurationEditorWindow):
+    def __init__(self, parent=None):
+        super(ConfigurationEditorWindow, self).__init__(parent)
+        self.setupUi(self)
+
+        self.frame.setStyleSheet("QFrame {border: 0;}")
+
+        settings_options = [
```

Comment:
we should use the "options" by the config class. We cannot have different names for those. Only names in the config class can be used. This may need a refactoring of them to have a grouping/categorizing. Also there is a need for a decription explaination which is then shown in the widget.

Reply:
I have just added this code for checking if the importing of the library works. I'm going to completely write this from scratch after finalizing the design mockup, which I haven't done yet.

Reply:
because of my network situation, I just added ideas ;)